src/shared-components/button/components/roundButton/index.tsx

```diff
@@ -77,6 +77,7 @@ const RoundButton = ({
     <RoundButtonWrapper>
       <RoundButtonBase
         onClick={!disabled && !isLoading ? onClick : () => false}
+        aria-disabled={disabled}
```

Comment:
ok so pardon my quick and opinionated investigation but this article: https://stackoverflow.com/questions/38059140/when-to-use-the-disabled-attribute-vs-the-aria-disabled-attribute-for-html-eleme

says we don't need the attribute when it is supported natively by the html component and yes we do need it when is a div for example. Looks like `RoundButtonBase` is a button do you think we still need the attribute for buttons?

Reply:
No pardon needed :sweat_smile: it looks like you're right about the buttons. I think I threw it on everything without much discernment, so I'm going to take a second look at this--we probably only need it for divs where we're passing `disabled` to apply styling. We want our accessibility tools to ignore all the content within that markup (which will still be in the DOM) in those cases. Thank you for the critical eye.